src/main/java/org/assertj/core/api/AbstractFileAssert.java

```diff
@@ -691,6 +692,15 @@ public SELF hasName(String expected) {
     return myself;
   }
 
+  /**
+   * Alias for {@link #hasName(String)}.
+   *
+   * This method can be used to easily switch between {@link File} and {@link Path} assertions.
+   */
+  public SELF hasFileName(String expected) {
+    return hasName(expected);
+  }
+
```

Comment:
- Have re-used `hasName` instead of adding a new method `assertHasFileName(info, actual, expected);` to `files` object.  Please let me know is this fine. 
- Please suggest a better javadoc if this does not explain it well 😉

Reply:
> * Have re-used `hasName` instead of adding a new method `assertHasFileName(info, actual, expected);` to `files` object.  Please let me know is this fine.

This is fine, thanks.

>     * Please suggest a better javadoc if this does not explain it well 😉

Answered in the other comment.